controls/scheduler/non-working-hours.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Non-working Hours
+page_title: .NET MAUI Scheduler Documentation - Non-working Hours
+description: Learn more about the non-working hours feature in the Telerik UI for .NET MAUI Scheduler control.
+published: False 
+slug: scheduler-non-working-hours
+---
+
+# Non - Working Hours
+
+The Telerik UI for .NET MAUI Scheduler control provides the option to define the day working hours of the work week in  the different Views. This means that you can set work start/end times, and in this case the remaining hours of the day will be styled differently, so they are noticeable across the timeline as non-working time.
+
+The control exposes a `IsWeekendVisible` property which defines a value that indicating whether the weekend days are visible, so that only the work week is displayed.
```

Comment:
```suggestion
The control exposes a `IsWeekendVisible` property which defines a value that indicates whether the weekend days are visible so that only the work week is displayed.
```